src/beerapi.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import datetime
+import time
+import logging
+
+def write_beer_inventory():
+    """
+    Retrieves inventory data from Tanczos and writes to file locally
+
+    Input:
+        Parameterless
+
+    Output:
+        Returns void
+    """
+    beer_inventory = requests.get('http://www.tanczos.com/tanczos.com/beerinventory/webexport.csv')
+    filename = str(datetime.datetime.now()).replace(' ', '_') + '.csv'
+    with open(filename, "w+") as beer_inventory_file:
+        beer_inventory_file.write(beer_inventory.text)
+
+def run():
+    """
+    Runs write_beer_inventory every 20 minutes
+
+    Input:
+        Parameterless
+
+    Output:
+        Returns void
+
+    Raises:
```

Comment:
No, this function does not raise an exception.  To say that a function raises an exception means that the caller must be prepared to handle the exception.  This documentation should be on the write_beer_inventory method. 

The documentation needs to explicitly state which exceptions can be thrown.